### ISSUE TITLE
AttributeError: module 'datetime' has not attribute 'UTC'

### ISSUE THREAD

**Issue by kentbull** [hyperledger-labs/did-webs-resolver#72]
When running the `get_started_docker.sh` script from within the `webs` container I get an AttributeError on UTC not being in datetime. 

Error stacktrace:
```python
Traceback (most recent call last):
  File "/keripy/venv/bin/dkr", line 8, in <module>
    sys.exit(main())
  File "/usr/local/var/webs/src/dkr/app/cli/dkr.py", line 33, in main
    raise ex
  File "/usr/local/var/webs/src/dkr/app/cli/dkr.py", line 30, in main
    directing.runController(doers=doers, expire=0.0)
  File "/keripy/src/keri/app/directing.py", line 665, in runController
    doist.do(doers=doers)
  File "/keripy/venv/lib/python3.10/site-packages/hio/base/doing.py", line 156, in do
    self.recur()  # increments .tyme runs recur context
  File "/keripy/venv/lib/python3.10/site-packages/hio/base/doing.py", line 275, in recur
    tock = dog.send(self.tyme)  # yielded tock == 0.0 means re-run asap
  File "/keripy/venv/lib/python3.10/site-packages/hio/base/doing.py", line 922, in do
    self.done = self.recur(tyme=tyme)  # equv of doist.recur
  File "/keripy/venv/lib/python3.10/site-packages/hio/base/doing.py", line 1026, in recur
    tock = dog.send(tyme)  # yielded tock == 0.0 means re-run asap
  File "/usr/local/var/webs/src/dkr/app/cli/commands/did/webs/generate.py", line 111, in generate
    result = didding.generateDIDDoc(self.hby, did=self.did, aid=aid, oobi=None, reg_name=self.da_reg, meta=self.meta)
  File "/usr/local/var/webs/src/dkr/core/didding.py", line 167, in generateDIDDoc
    retrieved=datetime.datetime.now(datetime.UTC).strftime(DID_TIME_FORMAT)
AttributeError: module 'datetime' has no attribute 'UTC'
```

This error causes the DID document generation to fail which halts the entire process.

**Comment by lemoustachiste** [comment]
Facing the same issue while following the getting started guide (https://github.com/hyperledger-labs/did-webs-resolver/blob/main/GETTING_STARTED.md) step by step.

**Comment by lemoustachiste** [comment]
tested and confirmed that @kentbull's fix in #73 resolves the issue
